js/popup.js

```diff
@@ -6,18 +6,37 @@ window.addEventListener('load', () => {
   let pageUrl = document.getElementById("page-url");
   let pageComment = document.getElementById("page-comment");
 
+  const findData = (items, url) => {
+    return items.data.filter((item) => item.url === url)[0] || null
```

Comment:
[IMO] この場合だと `find` で書き直すことができると思いますー


```suggestion
    return items.data.find((item) => item.url === url)[0]
```

そうすると関数にしなくてもよいかも...!